bin/local-env/install-wordpress.sh

```diff
@@ -88,7 +88,8 @@ container chmod 767 \
 echo -e $(status_message "Import default set of media assets...")
 # TODO: use glob pattern to import items. See https://developer.wordpress.org/cli/commands/media/import/.
 wp media import /var/www/html/wp-content/e2e-assets/small-video.mov
-wp media import /var/www/html/wp-content/e2e-assets/small-video.webm
+WEBM_VIDEO_ID=wp media import /var/www/html/wp-content/e2e-assets/small-video.webm --porcelain
+wp media import /var/www/html/wp-content/e2e-assets/small-video-poster.jpg --post_id=$WEBM_VIDEO_ID
```

Comment:
```suggestion
$POSTER_ID=wp media import /var/www/html/wp-content/e2e-assets/small-video-poster.jpg --post_id=$WEBM_VIDEO_ID --porcelain
wp post meta set $WEBM_VIDEO_ID _thumbnail_id $POSTER_ID
```

Reply:
Ah! Good call. Can actually use the `--featured_image` arg though, so don't need an extra line.

Reply:
```suggestion
wp media import /var/www/html/wp-content/e2e-assets/small-video-poster.jpg --post_id=$WEBM_VIDEO_ID --featured_image
``` 
So this? 

Didn't know that flag existed. Very cool

Reply:
Yep :) Just saw it in the docs too.